Update documentation for 0.3 release (#1473)
NVIDIA/spark-rapids: docs/download.md

---
layout: page
title: Download

> nav_order: 10

Reviewer: I suggest making this a much lower nav order, meaning it's much higher on the list and therefore easier to see.  Placing it last makes it look like we're trying to hide it and it's an afterthought.  Just after or before the "Getting Started" section seems like a logical order to me.

---

## Release v0.3.0

> This release includes additional performance improvements, for example, use of per thread default
> stream to make more efficient use of the GPU.  There is also further support for adaptive query
> execution, with more rewritten query plans now able to run on the GPU.  There are performance
> improvements for reading small Parquet files.  RAPIDS Shuffle with UCX has been updated to UCX
> 1.9.0.

Reviewer: Suggestion: bullet points seem apropos here, e.g.:
```suggestion
This release includes additional performance improvements, including:
- using per thread default stream to make more efficient use of the GPU
- supporting Spark's adaptive query execution, with more rewritten query plans now able to run on the GPU
- accelerating reads of many small Parquet files
- updating RAPIDS Shuffle with UCX to UCX 1.9.0
```

> Additional functionality includes Parquet reading for lists and structs, lead/lag for windows, and

Reviewer: Similarly may want a "new functionality" bullet list here.

greatest/least operators among others.

The release is supported on Apache Spark 3.0.0, 3.0.1, Databricks 7.3 ML LTS and Google Cloud
Platform Dataproc 2.0.

> GPU reading of cached dataframes is enabled but only effective for Spark 3.1, for those who

Reviewer: This requires the user configure the batched cache serializer, correct?  If we're going to mention it here we may want to point them to the documentation on how to do this.  AFAIK it won't "just work" if they use Spark 3.1 with the plugin.

wish to try out the latest version of Spark.

The list of all supported operations is provided [here](supported_ops.md).

For a detailed list of changes, please refer to the
[CHANGELOG](https://github.com/NVIDIA/spark-rapids/blob/main/CHANGELOG.md). 

Hardware Requirements: 

	GPU Architecture: NVIDIA Pascal™ or better (Tested on V100, T4 and A100 GPU)
	
Software Requirements:

> 	OS: Ubuntu 16.04 & gcc 5.4 OR Ubuntu 18.04/CentOS 7 & gcc 7.3

Reviewer: This project has no native code and therefore doesn't need a GCC version.
```suggestion
	OS: Ubuntu 16.04, Ubuntu 18.04, or CentOS 7
```

leverages columnar processing [SPARK-32332](https://issues.apache.org/jira/browse/SPARK-32332)
starting from Spark 3.0.1.

Another enhancement in v0.2 is improvement in reading small Parquet files.  This feature takes into
account the scenario where input data can be stored across many small files.  By leveraging multiple
CPU threads v0.2 delivers up to 6x performance improvement over the previous release for small
Parquet file reads.

The RAPIDS Accelerator introduces a beta feature that accelerates [Spark shuffle for
GPUs](get-started/getting-started-on-prem.md#enabling-rapidsshufflemanager).  Accelerated
shuffle makes use of high bandwidth transfers between GPUs (NVLink or p2p over PCIe) and leverages
RDMA (RoCE or Infiniband) for remote transfers. 

The list of all supported operations is provided
[here](configs.md#supported-gpu-operators-and-fine-tuning).

For a detailed list of changes, please refer to the
[CHANGELOG](https://github.com/NVIDIA/spark-rapids/blob/main/CHANGELOG.md). 

Hardware Requirements: 

	GPU Architecture: NVIDIA Pascal™ or better (Tested on V100, T4 and A100 GPU)
	
Software Requirements:

> 	OS: Ubuntu 16.04 & gcc 5.4 OR Ubuntu 18.04/CentOS 7 & gcc 7.3

Reviewer: ```suggestion
	OS: Ubuntu 16.04, Ubuntu 18.04, or CentOS 7
```

	
	CUDA & Nvidia Drivers: 10.1.2 & v418.87+, 10.2 & v440.33+ or 11.0 & v450.36+
	
	Apache Spark 3.0, 3.0.1
	
	Apache Hadoop 2.10+ or 3.1.1+ (3.1.1 for nvidia-docker version 2)
	
	Python 3.x, Scala 2.12, Java 8 

### Download v0.2.0
* [RAPIDS Spark Package](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/0.2.0/rapids-4-spark_2.12-0.2.0.jar)
* [cuDF 11.0 Package](https://repo1.maven.org/maven2/ai/rapids/cudf/0.15/cudf-0.15-cuda11.jar)
* [cuDF 10.2 Package](https://repo1.maven.org/maven2/ai/rapids/cudf/0.15/cudf-0.15-cuda10-2.jar)
* [cuDF 10.1 Package](https://repo1.maven.org/maven2/ai/rapids/cudf/0.15/cudf-0.15-cuda10-1.jar)

## Release v0.1.0

Hardware Requirements: 
   
    GPU Architecture: NVIDIA Pascal™ or better (Tested on V100 and T4 GPU)

Software Requirements: 

>     OS: Ubuntu 16.04 & gcc 5.4 OR Ubuntu 18.04/CentOS 7 & gcc 7.3

Reviewer: ```suggestion
    OS: Ubuntu 16.04 , Ubuntu 18.04, or CentOS 7
```
